wa/workloads/speedometer/__init__.py

```diff
@@ -25,6 +25,7 @@ class Speedometer(UiautoWorkload):
 
     name = 'speedometer'
     regex=re.compile(r'Speedometer Score ([\d.]+)')
+    versions = ['1.0', '2.0', '2.1']
```

Comment:
There doesn't seem to be a v2.1 of speedometer, I receive a Not Found page when trying to use that version.